src/System.Private.CoreLib/shared/System/Threading/SemaphoreSlim.cs

```diff
@@ -345,12 +345,11 @@ public bool Wait(int millisecondsTimeout, CancellationToken cancellationToken)
                     // spin, contention, etc. The usual number of spin iterations that would otherwise be used here is increased to
                     // lessen that extra expense of doing a proper wait.
                     int spinCount = SpinWait.SpinCountforSpinBeforeWait * 4;
-                    const int Sleep1Threshold = SpinWait.Sleep1ThresholdForSpinBeforeWait * 4;
 
                     var spinner = new SpinWait();
                     while (spinner.Count < spinCount)
                     {
-                        spinner.SpinOnce(Sleep1Threshold);
+                        spinner.SpinOnce(sleep1Threshold: -1);
```

Comment:
Should a similar change be made for other SpinOnce calls in CoreLib?  IIRC, this whole effort started because the spins in ConcurrentQueue were hitting Sleep(1)s... should those be changed here, too?

Reply:
Yea I'll do that in a separate set of changes across coreclr/corefx once the changes makes it to corefx